styles/variables-base.less

```diff
@@ -225,6 +227,7 @@
 // ---------------------------------------
 @agate-fanspeedcontrol-icon-font-size:   inherit;
 @agate-fanspeedcontrol-icon-height:      inherit;
+@agate-fanspeedcontrol-icon-line-height: 2rem;
```

Comment:
use pixel values in the code. Agate has a resolution independence system that will adapt this value to the screen size.